libs/ui/src/election_info_bar.tsx

```diff
@@ -24,7 +24,7 @@ export function ElectionInfoBar({
   precinctSelection,
 }: Props): JSX.Element {
   if (!electionDefinition) {
-    return <React.Fragment />;
+    return <React.Fragment>{null}</React.Fragment>;
```

Comment:
I don't love this, but `null` is not a `JSX.Element`, so we'd have to change all the `JSX.Element` return type annotations to be `React.ReactNode` instead.

Reply:
Can we just change the annotations as needed?

If I encountered this particular line of code I'd probably be confused.

Reply:
Actually, I tried a few different return type values and none of them made TypeScript happy:

- `React.ReactChild`: cannot return `null`, nor can the component be used as a JSX component.
- `React.ReactNode`: can return `null`, but cannot use the component as JSX component.
- `React.ReactElement`: cannot return `null`, but the component can be used as a JSX component.

I introduced a `Nothing` component in a new commit that should make this less confusing, anyway.

Reply:
Yeah, that's been my experience as well. I thought maybe I was wrong and `React.ReactNode` was working because you were using it here.

I do think returning `null` is idiomatic for React, so maybe we just make a new type? Something like: `type ReactComponent = React.ReactElement | null`? Or `type ReactComponent = ReturnType<React.FunctionalComponent>`?